RELEASE.md

```diff
@@ -7,32 +9,52 @@
 ### Notes
 
 - The `master` branch must refer to frontier `paritytech/frontier/master` or
-  `purestake/frontier/vX.X-hotfixes`
-
-### Pre-Release Checklist
-
-- [ ] Purge requirement has been communicated and scheduled with ops
-- [ ] Purge announcement has been prepared and scheduled
-- [ ] Documentation/Website/Tutorials have been updated
-- [ ] Runtime version has been updated
-- [ ] Tests are passing on StageNet
-- [ ] Ensure Type changes are reflected into PolkadotJs
+  `purestake/frontier/moonbeam-polkadot-vX.Y.Z`
 
 ### Release Workflow
 
+To release `vX.Y.Z`:
+
+1. Create a PR with increment client version
+1. Get that PR approved and merged
+1. Tag master with `vX.Y.Z` and push to github
+1. Start the github action "Publish Binary Draft" (on master branch)
+1. Review the generated Draft and clean a bit the messages if needed (keep it draft)
+1. Test the new client on internal tests networks (stagenet/moonsama/moonlama)
+1. Publish the client release draft
+1. When everything is ok, publish the new docker image: start github action Publish Docker with 
+`vX.Y.Z`
+1. Publish the new tracing image: on repo moonbeam-runtime-overrides, start github action
+Publish Docker with `vX.Y.Z` and master
+1. Documentation/Website/Tutorials have been updated
```

Comment:
are these supposed to be numbered like `1. 2. 3.` instead of `1. 1. 1.`

Reply:
same comment for further below

Reply:
Any markdown reader worth its salt will take care of the numbering in the rendering: https://github.com/PureStake/moonbeam/blob/elois-doc-release/RELEASE.md

Reply:
Cool, didn't know this worked like that